extraction_tools_wiki/wiki_page_text.py

```diff
@@ -58,37 +58,44 @@ def extract_page_wiki_text(self):
         request['format'] = 'json'
         request['page'] = self.page_title
 
-        page_data = requests.get(self.base_url,
-                                 headers=self.custom_agent,
-                                 params=request).json()
+        # Perform HTTP GET request
+        try:
+            page_data = requests.get(self.base_url,
+                                     headers=self.custom_agent,
+                                     params=request).json()
+        except requests.exceptions.RequestException as e:
+            print(e)
+            sys.exit(">>> ERROR: Get request error. Exiting.")

```

Comment:
So a cool way to maintain the stack trace and exit is to use a newer python feature `raise from`. It takes the exceptions stacktrace and saves it in your new exception (we will throw `SystemExit`)

```python
        # Perform HTTP GET request
        try:
            page_data = requests.get(self.base_url,
                                     headers=self.custom_agent,
                                     params=request).json()
        except requests.exceptions.RequestException as e:
            raise SystemExit("Request error!") from e
```